### ISSUE TITLE
zarf-lib: expose the cluster in the packager

### ISSUE THREAD

**Issue by UncleGedd** [zarf-dev/zarf#2977]
### Is your feature request related to a problem? Please describe.

We're using Zarf as a library, and want to know which cluster a package/component is using for the deploy, and would like to be able to retrieve the[ k8s client](https://github.com/zarf-dev/zarf/blob/efa306fdcbe605ec6d17f4cf72c707f7227af52a/src/pkg/packager/common.go#L141) that Zarf is using

### Describe the solution you'd like

Either make the `cluster` field in `Packager` public or provide a getter

**Comment by mkcp** [comment]
Appreciate the ask @UncleGedd! I believe this is blocked by the packager refactor in https://github.com/zarf-dev/zarf/issues/3525?issue=zarf-dev%7Czarf%7C3528. Once we migrate Deploy behavior over to our the new standards for the SDK we can better address your usecase.

**Comment by nick-adkins** [comment]
@UncleGedd is this still a relevant issue?

**Comment by AustinAbro321** [comment]
Closing this as the structure of the SDK has changed significantly since this issue was opened. Feel free to re-open another issue if there is still a need
